poh/src/poh_recorder.rs

```diff
@@ -395,23 +438,19 @@ impl PohRecorder {
         self.leader_last_tick_height = leader_last_tick_height;
     }
 
-    pub fn set_working_bank(&mut self, working_bank: WorkingBank) {
-        trace!("new working bank");
-        assert_eq!(working_bank.bank.ticks_per_slot(), self.ticks_per_slot());
-        self.working_bank = Some(working_bank);
-        // TODO: adjust the working_bank.start time based on number of ticks
-        // that have already elapsed based on current tick height.
-        let _ = self.flush_cache(false);
-    }
-
```

Comment:
This functionality was only used over `set_bank()` in tests, which have been refactored to avoid using this function. Thus this function can now be deleted